### PR TITLE
NNS1-2851: Refactor uncertifiedLoadAccountsBalance

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Unify wallet implementations.

In this PR, refactor `uncertifiedLoadAccountsBalance` to use current services instead of creating new ones.

# Changes

* Change implementation of `uncertifiedLoadAccountsBalance`.

# Tests

Same functionality.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4445** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4445?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4445?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/services/wallet-uncertified-accounts.services.ts [hunk 1]
```diff
@@ -82,8 +28,14 @@ export const uncertifiedLoadAccountsBalance = async ({
         ) ?? []
       ).map((universeId) =>
         Promise.all([
-          loadAccountsBalance(Principal.fromText(universeId)),
-          loadToken(Principal.fromText(universeId)),
+          loadAccounts({
+            // TODO: Use strategy `"query"`
```
Comment (dskloetd) [inline frontend/src/lib/services/wallet-uncertified-accounts.services.ts:32]: What's preventing you from doing this already?
And without this, would this actually be uncertified, like the name of the function claims?

Comment (lmuntaner) [reply frontend/src/lib/services/wallet-uncertified-accounts.services.ts]: Yes, I knew I would forget about it and that's why I added the todo.

Done.

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**